src/main/scala/com/ing/wbaa/airlock/proxy/provider/atlas/LineageHelpers.scala

```diff
@@ -141,11 +148,22 @@ trait LineageHelpers extends LazyLogging with RestClient {
       bucketGuid <- lineageGuidFuture(lh.bucket, AWS_S3_BUCKET_TYPE, bucketEntities(lh.bucket, AIRLOCK_PII).toJson).map(_.entityGUID)
       pseudoDirGuid <- lineageGuidFuture(pseudoDir, AWS_S3_PSEUDO_DIR_TYPE, pseudoDirEntities(bucketGuid, pseudoDir).toJson).map(_.entityGUID)
       objectGuid <- lineageGuidFuture(bucketObject, AWS_S3_OBJECT_TYPE, bucketObjectEntities(pseudoDirGuid, bucketObject, lh.contentType, AIRLOCK_PII).toJson).map(_.entityGUID)
+      fsPathGuid <- externalFsPath match {
+        case Some(fsPath) => lineageGuidFuture(fsPath, HADOOP_FS_PATH, fsPathEntities(fsPath).toJson).map(_.entityGUID)
+        case None         => Future("")
```

Comment:
```suggestion
        case None         => Future.successful("")
```
if faster when if the argument is computed